concordium-consensus/src/Concordium/GlobalState/Basic/BlockState/Account.hs

```diff
@@ -40,6 +42,24 @@ makeAccountPersisting :: PersistingAccountData -> AccountPersisting
 makeAccountPersisting = makeHashed
 {-# INLINE makeAccountPersisting #-}
 
+-- | The account token table, with all references loaded. TODO (drsk) I'd like
+-- to move this to Persistent/GlobalState/ProtocolLevelTokens.hs, but this
+-- would result in cyclig imports.
+newtype InMemoryTokenStateTable = InMemoryTokenStateTable
```

Comment:
This should find a new place and if possible reuse the hashing already defined in `ProtocolLevelTokens.hs`.

Reply:
This has been moved. But there is a discussion about it in a later comment.